oauth2client/service_account.py

```diff
@@ -461,3 +465,94 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+def _datetime_to_secs(utc_time):
+   # time_delta.total_seconds() not supported in Python 2.6
+   epoch = datetime.datetime(1970, 1, 1)
+   time_delta = utc_time - epoch
+   return time_delta.days * 86400 + time_delta.seconds
+
+
+class JWTAccessCredentials(GoogleCredentials):
+    """Self signed JWT credentials.
+
+    Makes an assertion to server using a self signed JWT from service account
+    credentials.  These credentials do NOT use OAuth 2.0 and instead authenticate
+    directly.
+    """
+    MAX_TOKEN_LIFETIME_SECS = 3600
+    """Max lifetime of the token (one hour, in seconds)."""
+
+    def __init__(self, service_account_creds, additional_claims=None):
+        super(JWTAccessCredentials, self).__init__(None, None, None, None, None,
+                                                   None, None)
+        self._creds = service_account_creds
+        self._claims = {}
+        if additional_claims:
+            self._claims.update(additional_claims)
+    
+    def get_access_token(self, http=None, additional_claims=None):
+        """Create a signed jwt
+        Agrs:
+            additional_claims: dict, additional claims to add 
+                the payload of the JWT. 
+            Returns:
+                An AccessTokenInfo with the signed jwt
+        """
+        if additional_claims is not None:
+            # Create a 1 time token
+            token, expiry = self._create_token(additional_claims)
```

Comment:
`_` is allowed; I counsel you to use `unused_<helpfully descriptive name>` as a kindness to the poor soul in the future learning the codebase.